spec/cli/test/agent-only.spec.js

```diff
@@ -42,7 +42,7 @@ describe('TestCommand test suite for agent-only scenario', () => {
   it('should run a command', (done) => {
     run({
       configPath:  '/fixtures/agent-only/.imptest',
-      testCaseFile:  '/fixtures/agent-only/tests/agent.test.nut'
+//      testCaseFile:  '/fixtures/agent-only/tests/agent.test.nut'
```

Comment:
This is old way to set test file pattern in command line. In current version the test file pattern is set in config file. The individual test is defined in CLI only. So testCaseFile:  '/fixtures/agent-only/tests/agent.test.nut' is interpret as "/fixtures/agent-only/tests/agent" class and "test.nut" method. The '/fixtures/agent-only/tests/agent.test.nut' test file can be found by file pattern in config file of the agent-only test.